src/main/java/kdt/prgrms/kazedon/everevent/service/UserService.java

```diff
@@ -43,6 +48,16 @@ public void checkNicknameDuplicate(String nickname) {
     }
   }
 
+  @Transactional
+  public UserType changeAuthorityToBusiness(String email) {
+    User user = userRepository.findByEmail(email)
+        .orElseThrow(() -> new NotFoundException(ErrorMessage.USER_NOT_FOUNDED, email));
+    authorityRepository.deleteByUserId(user.getId());
+    user.changeAuthority(UserType.ROLE_BUSINESS);
+    Authority authority = userRepository.save(user).getAuthority().get(0);
```

Comment:
```suggestion
    Authority authority = userRepository.save(user).getAuthority().get(0);
```
이부분을 조금 풀수 없을까요?

https://mangkyu.tistory.com/147
디미터의 법칙에도 위반한거 같기도 하네요.

Reply:
넵 잘못된 것 같습니다ㅠㅠ authority를 테이블로 관리하면서 생긴 문제 같아서 테이블이 아닌 User 테이블의 컬럼으로 저장하는걸로 수정할 예정입니다!